geode-core/src/main/java/org/apache/geode/management/internal/AgentUtil.java

```diff
@@ -91,6 +94,13 @@ public String findWarLocation(String warFilePrefix) {
     return null;
   }
 
+  public List<String> lookupWarLocationFromClasspath(String warFilePrefix) {
```

Comment:
This should be private. I think the test should be using `findWarLocation` instead.

Reply:
I thought of that, but I wanted to specifically test the method to lookup the war from the classpath. But I can change the test to use the `findWarLocation`

Reply:
If we do this, do we still need code block 79 to 90?

Reply:
@jinmeiliao ... currently looking at this code block myself... possibly this block could be replaced.. I would still like to find the code that is dependent on this.